docs/user-guide/admin-cli.md

```diff
@@ -315,90 +389,95 @@ optional arguments:
+[root@localhost:~]
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access add --name=vmgroup1 --datastore=datastore2  --volume-maxsize=500MB --volume-totalsize=1GB
 vmgroup access add succeeded
-
+[root@localhost:~]
 [root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access ls --name=vmgroup1
 Datastore   Allow_create  Max_volume_size  Total_size
 ----------  ------------  ---------------  ----------
-datastore1  False         500.00MB         1.00GB
-datastore2  True          500.00MB         1.00GB
-
-[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup ls
-Uuid                                  Name       Description                 Default_datastore  VM_list
-------------------------------------  ---------  --------------------------  -----------------  -------
-11111111-1111-1111-1111-111111111111  _DEFAULT   This is a default vmgroup
-6d810c66-ffc7-47c8-8870-72114f86c2cf  vmgroup1                              datastore2         photon7
+datastore1  True          Unset            Unset
+datastore2  False         500.00MB         1.00GB
 
 ```
 
 By default no "allow_create" right is given
 
 ```bash
-[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access add --name=vmgroup1 --datastore=datastore1  --volume-maxsize=500MB --volume-totalsize=1GB
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access add --name=vmgroup1 --datastore=datastore2  --volume-maxsize=500MB --volume-totalsize=1GB
 vmgroup access add succeeded
-
+[root@localhost:~]
 [root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access ls --name=vmgroup1
 Datastore   Allow_create  Max_volume_size  Total_size
 ----------  ------------  ---------------  ----------
-datastore1  False         500.00MB         1.00GB
+datastore1  True          Unset            Unset
+datastore2  False         500.00MB         1.00GB
 ```
 
 "allow_create" right is given when you run the command with "--allow-create" flag.
 ```bash
-[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access add --name=vmgroup1 --datastore=datastore2  --allow-create --default-datastore --volume-maxsize=500MB --volume-totalsize=1GB
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access add --name=vmgroup1 --datastore=datastore2  --volume-maxsize=500MB --volume-totalsize=1GB --allow-create
 vmgroup access add succeeded
-
+[root@localhost:~]
+[root@localhost:~]
 [root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access ls --name=vmgroup1
 Datastore   Allow_create  Max_volume_size  Total_size
 ----------  ------------  ---------------  ----------
-datastore1  False         500.00MB         1.00GB
+datastore1  True          Unset            Unset
 datastore2  True          500.00MB         1.00GB
 ```
 
+"-volume-totalsize" cannot be set to the value other than unlimit when add privilege for special value "__VM_DS" and "__ALL_DS".
```

Comment:
what does it mean ? I am confused. Are you saying "for _VM_DS and _ALL_DS special DS names, --volume-totalzie has to be set to "umlimit", or command will fail"   ?